monash-human-power/raspicam
# Update topic structure for camera status messages

## Description

The topic structure (and variable name) was quite verbose and inconsistent. Has been changed as specified in MQTT V3 Topics for use in [dashboard#77](https://github.com/monash-human-power/dashboard/pull/77).

## Steps to Test

1. Open `topics.py`

2. Open the MQTT V3 Topics page

3. Observe that the changed topics conform to the documentation
